ideas/170-wallet-improvements.md

```diff
@@ -0,0 +1,74 @@
+## Preamble
+
+    Idea: 170
+    Title: Wallet Improvements
+    Status: Draft
+    Created: 2018-04-13
+    
+## Summary
+
+Status Wallet does not fully support ERC20 tokens (requesting and transaction history not available)
+and generally the transactions UX is different between Chat and Wallet. The intention of this swarm
+is to wrap up those inconsistencies.
```

Comment:
and perhaps add "and study and improve wallet and transaction UX problems and limitations to make the wallet experience great"?